circle.yml

```diff
@@ -92,6 +92,53 @@ jobs:
       - build
       - test
 
+  clang-latest-coverage:
+    executor: linux-clang-9
+    environment:
+      BUILD_TYPE: Coverage
+      TESTS_FILTER: unittests
+    steps:
+      - run:
+          name: "Install lcov"
+          working_directory: /tmp
+          command: |
+            sudo apt-get -q update
+            # sudo apt-get install -qy lcov --no-install-recommends
+            apt-get download lcov
+            sudo dpkg -i --force-depends lcov*.deb
+      - build
+      - test
+      - run:
+          name: "Collect coverage data"
+          working_directory: ~/build
+          command: |
+            mkdir coverage
+            find -name '*.profraw'
+            llvm-profdata merge *.profraw -o fizzy.profdata
+
+            llvm-cov report -instr-profile fizzy.profdata -Xdemangler llvm-cxxfilt \
```

Comment:
Output of this line doesn't seem to be saved anywhere?

Reply:
It is meant for CI log.